tests/test_common/test_fixture.cpp

```diff
@@ -58,7 +58,8 @@ void TestFixture::TearDownTestCase() {}
 TestFixture::TestFixture() {
     m_this = this;
     timer_clear();
-    test_logger.info() << "tapping term is " << +GET_TAPPING_TERM(KC_TRANSPARENT, &(keyrecord_t){}) << "ms" << std::endl;
+    keyrecord_t empty_keyrecord = {};
+    test_logger.info() << "tapping term is " << +GET_TAPPING_TERM(KC_TRANSPARENT, &empty_keyrecord) << "ms" << std::endl;
```

Comment:
This change is unrelated to housekeeping, but an issue I ran into on my personal forked repo.  The following error occurs when `TAPPING_TERM` is defined for a test:

![image](https://github.com/qmk/qmk_firmware/assets/66687468/1e2347e9-f134-4f83-a922-85f9bfbd169e)

This change fixes that.  I'm happy to put this in a separate CR, but it was small enough that I figured it was fine to include here.